try {
  console.log(aasasas);
} catch (error) {
  console.log("erro ");
}
